package.json

```diff
@@ -14,11 +14,14 @@
   "scripts": {
     "start": "node tools/server.js",
     "build": "NODE_ENV=production node tools/build.js && npm run build-js && npm run minify-js",
+    "build-w": "SET NODE_ENV=production node tools/build.js & npm run build-js-w & npm run minify-js-w",
```

Comment:
Would be great if there was a way not to have two separate commands - or if you stopped using windows :)